debian/rules

```diff
@@ -52,7 +52,7 @@ DEB_DH_INSTALL_ARGS_$(REFDBG_PKG) += --sourcedir=debian/install/refdbg
 
 DEB_DH_MAKESHLIBS_ARGS_$(SHARED_PKG) += -V --add-udeb=$(UDEB_PKG) -- -c4
 DEB_DH_MAKESHLIBS_ARGS_$(REFDBG_PKG) = --no-act
-DEB_DH_STRIP_ARGS_$(REFDBG_PKG) = --no-act
```

Comment:
Have you tried building? We used to have an old version of something that was preventing us from using that flag.

https://github.com/endlessm/glib/commit/5bd5c564db0c04ad808f7766b386ea1078bfa2b3